lib/rules/alpha-value-notation/index.cjs

```diff
@@ -102,9 +104,18 @@ const rule = (primary, secondaryOptions) => {
 				const unfixed = value;
 				const index = declarationValueIndex(decl) + alpha.sourceIndex;
 				const endIndex = index + alpha.value.length;
-				const fix = () => {
-					alpha.value = fixed;
-					setDeclarationValue(decl, parsedValue.toString());
+
+				/** @type {Problem['fix']} */
```

Comment:
Unless you pass `fix` directly to `report`, with the proposal 3, you will have to add the type for every declaration. That's a big CON but not a blocker.